sdks/java/io/file-based-io-tests/pom.xml

```diff
@@ -31,6 +31,112 @@
     <name>Apache Beam :: SDKs :: Java :: IO :: File-based-io-tests</name>
     <description>Integration tests for reading/writing using file-based sources/sinks.</description>
 
+    <profiles>
+        <!-- Include the Google Cloud Dataflow runner activated by -DintegrationTestRunner=dataflow -->
+        <profile>
+            <id>dataflow-runner</id>
+            <activation>
+                <property>
+                    <name>integrationTestRunner</name>
+                    <value>dataflow</value>
+                </property>
+            </activation>
+            <dependencies>
+                <dependency>
+                    <groupId>org.apache.beam</groupId>
+                    <artifactId>beam-runners-google-cloud-dataflow-java</artifactId>
+                    <scope>runtime</scope>
+                </dependency>
+            </dependencies>
+        </profile>
+
+        <!--
+            This profile invokes PerfKitBenchmarker, which does benchmarking of
+            the IO ITs. The arguments passed to it allow it to invoke mvn again
+            with the desired benchmark.
+
+            To invoke this, run:
+
+            mvn verify -Dio-it-suite -pl sdks/java/io/file-based-io-tests
```

Comment:
How about mentioning instructions for running using DirectRunner and referring to I/O IT doc for other runners ?

Reply:
I followed JDBC case, but no problem - I'll modify it for DirectRunner

Reply:
please ignore the comment above, I misunderstood you. check it now - I hope it fits into your idea.

Reply:
Yeah, this looks good.